Contributions/README.md

```diff
@@ -5,25 +5,28 @@ compatible licenses, and peer-reviewed derived work, produced under the
 ELISA umbrella.
 
 ## Index
-* [Interference Scenarios for an ARM64 Linux System](Interference_Scenarios_for_an_ARM64_Linux_System.pdf)
-  (3rd party) Analysis of a reference system based on ARM64, running Linux.
-  It provides architectural elements, both SW and HW, which are then used
-  as reference for describing examples of interference that the system is
-  subject to.
+These markdown documents are actively extended and reviewed, as part of safety-related work done within ELISA.
+
+* [CheckList for Safety Claims on a Generic Linux System](CheckList_for_Safety_Claims_on_a_Generic_Linux_System.md)
+  (3rd party) Non-exhaustive list of safety woes that can affect one's system qualification and should be explicitly addressed, in preparation to the qualification process.
```

Comment:
At what point would this change from `(3rd party)` to `(ELISA)`?

Reply:
Me being the collaborator, I'd say I'm the least qualified to answer the question coming from an ELISA WG Chair :-D

Reply:
To clarify: AFAIK my contributions are not tagged ELISA until someone who can say so does claim that indeed they are approved by ELISA. Here I simply rectified my previous mistake in not stating out clearly the status of the document.

Reply:
Considering the dependency between documents, first the one about FFI should be vetted, then the one about requirements, and only then it would be possible to review and promote this one.